index.js

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+const yargs = require('yargs/yargs');
+
+const { argv } = yargs(process.argv.slice(2))
+    .coerce('_', args => {
+        return args.map(arg => {
+            const num = Number(arg);
+            if (isNaN(num)) {
+                console.error('Numbers must be provided and must be numeric');
+                process.exit(1)
+            }
+            return num;
+        })
+
+    })
+    .usage('Usage: --operation <operation name> <values>')
+    .example("node index.js --operation addition 1 2 3","\nReturns Result = 6")
+    .option("operation", {
+        describe: "Operations available", type: "string",
+        choices: ['addition', 'subtraction', 'multiply', 'division'],
+        demandOption: true
+    })
+
+
+printWelcome();
+const operation = argv.operation;
+const operand = argv._;
+
+
+if (operation === "addition") {
```

Comment:
const OPERATIONS = {
   ADD: 'addition',
  SUB: 'subscr'....
}

and replace everywhere where we're using this string.